docs/webhook-config.md

```diff
@@ -80,6 +80,26 @@ When using the Form-Encoded or JSON-Encoded configuration you can configure any
 
 The result would be a POST request with e.g. `Status: running` body and `Content-Type: text/plain` header.
 
+### Example: WeCom Webhook Configuration
+
+In addition to generic JSON formatting, `webhook` now supports payloads compatible with enterprise messaging tools such as **WeCom (WeChat Work)** and **DingTalk group bots**.
```

Comment:
An example webhook configuration for this is fine, including the sample below on what the recepient would expect.

I'd however want to remove the reference to specific products - as it makes it look like we actively support or recommend these - which we don't.

---

also the "now" wording is wrong - it'll stay in the docs most likely forever (unless it's buggy). this wording fits a changelog - but not docs.